backend/src/main/java/com/huellapositiva/application/controller/VolunteerApiController.java

```diff
@@ -50,6 +54,14 @@
 
     private final UploadCurriculumVitaeAction uploadCurriculumVitaeAction;
 
+    private final FetchVolunteerProfileAction fetchVolunteerProfileAction;
+
+    private final UpdateVolunteerProfileAction updateVolunteerProfileAction;
+
+    private final UploadPhotoAction uploadPhotoAction;
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Reuse as a Bean

Reply:
I don`t use the Object Mapper in this class, but i go to create a card in trello about this and we will refactor whole the project.